components/windows/swap-widget.tsx

```diff
@@ -277,6 +279,14 @@ export const SwapWidget = () => {
                         image="/windows/swap/ETH.svg"
                         maxBalance={formattedEthBalance}
                         setFocused={setFocused}
+                        calculatePriceAndToken={() =>
```

Comment:
much better to not use arrow functions in props but rather creating handler functions above jsx.. this makes the whole component much easier to read and uses less compute e.g making app more performant

Reply:
Done